### DISCUSSION TITLE
Button Call InputActionEvent

### DISCUSSION THREAD

**Discussion by KC098** [godotengine/godot-proposals#4155] [GUI]
Why is it not possible to call for an InputActionEvent on Control Buttons? TouchScreenButton does that so why not integrate that function?

Because I'm developing an app.
The current problem I'm facing with this is that is it not possible (out of the box) to do multitouch with control nodes you can program a work around but is more complicated than that. You have to make sure the touch is detected inside the control, stop touch when there is an active control above. Among other things.

**Comment by SilencedPerson** [comment]
I am pretty sure this was already proposed.
